libpimeval/src/pimParamsPerf.cpp

```diff
@@ -576,6 +583,9 @@ pimParamsPerf::getMsRuntimeBitSerial(PimDeviceEnum deviceType, PimCmdEnum cmdTyp
         if (it3 != it2->second.end()) {
           const auto& [numR, numW, numL] = it3->second;
           msRuntime += m_tR * numR + m_tW * numW + m_tL * numL;
+          mjEnergy += m_eR * numR + m_eL *numL;
```

Comment:
Do we consider numW also?

Reply:
Activate Energy and Precharge Energy are combined in m_eR, that's also the reason that we don't know the energy number for a separate 'read' or 'write'